extensions/2.0/Vendor/EXT_feature_metadata/examples/weather/weather.gltf

```diff
@@ -0,0 +1,219 @@
+            },
+            "windVelocity": {
+              "valueType": "FLOAT32",
+              "elementType": "ARRAY",
+              "valueCount": 3
+            }
+          }
+        },
+        "town": {
+          "name": "Town Data",
+          "properties": {
+            "name": {
+              "name": "Name",
+              "valueType": "STRING"
+            },
+            "population": {
+              "name": "Population",
+              "valueType": "FLOAT32"
+            }
+          }
+        }
+      },
+      "featureCollections": {
+        "weatherCollection": {
+          "featureClass": "weather",
+          "featureCount": 1000,
+          "properties": {
+            "airTemperature": {
+              "elementCount": 1000,
+              "bufferView": 2
+            },
+            "airPressure": {
+              "elementCount": 1000,
+              "bufferView": 3
+            },
+            "windVelocity": {
+              "elementCount": 1000,
+              "bufferView": 4
+            }
+          }
+        },
+        "townCollection": {
+          "featureClass": "town",
+          "featureCount": 3,
+          "properties": {
+            "name": {
+              "elementCount": 3,
+              "bufferView": 5,
+              "offsetsBufferView": 6
+            },
+            "population": {
+              "elementCount": 3,
+              "bufferView": 7
+            }
+          }
+        }
+      }
+    }
+  },
+  "scene": 0,
+  "scenes": [
+    {
+      "name": "Scene",
+      "nodes": [
+        0
+      ]
+    }
+  ],
+  "nodes": [
+    {
+      "name": "Point Cloud",
+      "mesh": 0
+    }
+  ],
+  "meshes": [
+    {
+      "name": "Point Cloud Mesh",
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 0,
+            "_FEATURE_ID_0": 1
+          },
+          "mode": 0,
+          "extensions": {
+            "EXT_feature_metadata": {
+              "featureMappings": [
+                {
+                  "featureCollection": "weatherCollection",
+                  "featureIds": {
+                    "constant": 0,
+                    "vertexStride": 1
+                  }
+                },
+                {
+                  "featureCollection": "townCollection",
+                  "featureIds": {
+                    "attribute": "_FEATURE_ID_0"
+                  }
+                }
+              ]
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "accessors": [
+    {
+      "name": "Positions",
+      "bufferView": 0,
+      "byteOffset": 0,
+      "componentType": 5126,
+      "count": 1000,
+      "max": [
+        0.9984954469408285,
+        0.9919821104589452,
+        0.9970798239119962
+      ],
+      "min": [
+        -0.9968233643517341,
+        -0.9973561515575708,
+        -0.9991830031102058
+      ],
+      "type": "VEC3"
+    },
+    {
+      "name": "Town Feature ID",
+      "bufferView": 1,
```

Comment:
It's debatable, but I kinda think feature IDs belong in the 0th buffer with other vertex data.

Reply:
We discussed this, we decided it's best to let the user determine where to store the feature IDs:

* if the model was created in Blender/Maya/etc. and then feature metadata was tacked on, using a separate buffer makes sense
* If the model was created programmatically, using the 0th buffer might make more sense (still debatable)